pkgs/development/python-modules/sendgrid/default.nix

```diff
@@ -11,7 +11,7 @@
 
 buildPythonPackage rec {
   pname = "sendgrid";
-  version = "6.7.0";
+  version = "6.7.1";
 
   src = fetchFromGitHub {
     owner = pname;
```

Comment:
Missing hash change.

Reply:
Thanks a lot for the feedback. Hash updated.